resources/js/admin/settings/components/SettingsMenu.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div
+    class="d-flex"
+  >
+    <div
+      id="menu"
+      class="menu"
+    >
+      <settings-menu-collapse
+        @selectGroup="selectGroup"
+      />      
+    </div>
+    <div class="setting-info">
+      <settings-listing
+        :group="group"
+        @refresh="refresh"
+        @refresh-all="refreshAll"
+        ref="listings"
+      />
+      </div>
+  </div>
+</template>
+
+<script>
+import SettingsListing from './SettingsListing';
+import SettingsMenuCollapse from './SettingsMenuCollapse';
+
+export default {
+  components: { SettingsListing, SettingsMenuCollapse },
+  data() {
+    return {
+      currentTab: 0,
+      group: "Action by Email",
```

Comment:
Please not use a harcoded value, what happen in the Action by Email label was renamed?
Maybe the current value selected is the firstOption from the firstMenu